### PR TITLE
MINOR: fix duplicated return and other streams docs typos

### PR DESCRIPTION
**Pull request by AyoubOm**

### CODE REVIEW

File: streams/src/main/java/org/apache/kafka/streams/kstream/NamedOperation.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 package org.apache.kafka.streams.kstream;
 
 /**
- * Default interface which can be used to personalized the named of operations, internal topics or store.
+ * Default interface which can be used to customize the named of operations, internal topics or store.
```
Comment (johnnychhsu) [inline streams/src/main/java/org/apache/kafka/streams/kstream/NamedOperation.java]: shall we also change the description to 'the name of operations'?

**Comment by johnnychhsu** [comment]
thanks for the pr!
this looks good to me, leave a simple comment

File: streams/src/main/java/org/apache/kafka/streams/kstream/NamedOperation.java (hunk 1 above)
Comment (AyoubOm) [reply streams/src/main/java/org/apache/kafka/streams/kstream/NamedOperation.java]: Thanks @johnnychhsu, fixed.

**Comment by AyoubOm** [comment]
@chia7712 please check this when you have time

**Review by chia7712 — APPROVED** [review]
LGTM

**Comment by chia7712** [comment]
@AyoubOm thanks for your contribution

**Pull request merged**